README.md

```diff
@@ -216,7 +216,10 @@ You can customize the name of the `.key` property by passing an option when init
 
 ```javascript
 require('firebase/firestore')
-Vue.use(VueFirestore, {key = 'id'})
+Vue.use(VueFirestore, {

```

Comment:
Can you change this to 
`{ key: 'id', enumerable: false }`instead of `key = 'id'` and `enumerable = false`